src/test/java/Homework17A.java

```diff
@@ -0,0 +1,43 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework17A extends MyBaseTest{
+//    public static WebDriver driver = new ChromeDriver();
+//    we can remove this webdriver initialization
+
+
+    @Test
+    public void addSongtoPlaylist() throws InterruptedException {
+        //login("simplyd2d@gmail.com","te$t$tudent");
+    enterEmail("simplyd2d@gmail.com");
+    enterPassword("te$t$tudent");
+    clickSubmit();
+    Thread.sleep(2000);
+
+    searchSong("Take My Hand");
+    Thread.sleep(2000);
+    viewAllSearchResults();
+    selectSong();
+    clickAddToButton();
+    selectPlaylistname("David");
+
+       WebElement addedSongAlert = driver.findElement(By.cssSelector("div.success.show"));
+        Assert.assertTrue(addedSongAlert.isDisplayed());
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)